lib/response_massager.js

```diff
@@ -19,8 +18,11 @@ class ResponseMassager {
    *
    *  Also copies ".total" properties into convenient places for serialization.
    */
-  processInnerHitsProperties (response) {
+  processInnerHitsProperties (response, allItemsBibQuery) {
```

Comment:
I hear you. I think it's fine on balance. This seems like the right place to sort items. What triggers the sorting is the question. An alternative to adding a "do the sorting" param would be to change the check on line 23 to `if (hit._source.items.some((i) => i.enumerationChronology_sort))` since items will only have that property when we've specifically requested it for manual sorting. But what you've done is prob clearer.